gitpod-io/gitpod
# [protocol] ContextURL.parse

This adds code that is able to parse our `contextURL` field into an actual `URL`.

This came up here: https://github.com/gitpod-io/gitpod/pull/4262#issuecomment-857485960

#### Test
 - start prebuild: https://gpl-context-url-parser.staging.gitpod-dev.com/#prebuild/github.com/gitpod-io/gitpod/tree/gpl/test-prebuild (note the missing scheme in the URL)
 - check that the prebuild starts and the frontend streams logs successfully
